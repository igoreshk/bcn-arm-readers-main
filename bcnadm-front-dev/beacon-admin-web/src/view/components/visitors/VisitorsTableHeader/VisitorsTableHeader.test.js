import React from 'react';
import { shallow } from 'enzyme';
import { VisitorsTableHeader } from './index';

const props = {
  translate: jest.fn(),
  toggleFilter: jest.fn(),
  toggleSortVisitorsByField: jest.fn()
};

const wrapper = shallow(<VisitorsTableHeader {...props} />);

describe('VisitorsTableHeader', () => {
  it('Check matching snapshot', () => expect(wrapper).toMatchSnapshot());
});
