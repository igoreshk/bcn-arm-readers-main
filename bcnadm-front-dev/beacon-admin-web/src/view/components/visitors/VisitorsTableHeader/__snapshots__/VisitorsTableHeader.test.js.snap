// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisitorsTableHeader Check matching snapshot 1`] = `
<WithStyles(ForwardRef(TableRow))>
  <WithStyles(ForwardRef(TableCell))>
    <div
      onClick={[Function]}
      style={
        Object {
          "alignItems": "center",
          "color": "rgba(0, 0, 0, 0.54)",
          "cursor": "pointer",
          "display": "flex",
        }
      }
    >
      <Unsorted />
    </div>
  </WithStyles(ForwardRef(TableCell))>
  <WithStyles(ForwardRef(TableCell))
    className="remove"
  >
    <WithStyles(ForwardRef(Button))
      className="filterButton"
      onClick={[MockFunction]}
    >
      <Memo(ForwardRef(FilterListIcon)) />
      <span
        style={
          Object {
            "marginLeft": "10px",
          }
        }
      />
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(TableCell))>
</WithStyles(ForwardRef(TableRow))>
`;
